.circleci/config.yml

```diff
@@ -84,12 +84,12 @@ jobs:
               echo "Skipping snapshot deployment: wrong JDK. Expected '$JDK' but was '$CIRCLE_JDK_VERSION'."
             else
               echo "Deploying snapshot..."
-              ./gradlew androidSourcesJar androidJavadocsJar uploadArchives \
+              ./gradlew androidSourcesJar androidJavadocsJar publish closeAndReleaseRepository \
               -Psigning.keyId=$GPG_SIGNING_KEY_ID \
               -Psigning.password=$GPG_SIGNING_KEY_PW \
               -Psigning.secretKeyRingFile=./secring.gpg \
-              -PSONATYPE_NEXUS_USERNAME=$SONATYPE_NEXUS_USERNAME \
-              -PSONATYPE_NEXUS_PASSWORD=$SONATYPE_NEXUS_PASSWORD \
+              -PmavenCentralUsername=$SONATYPE_NEXUS_TOKEN_USERNAME \
```

Comment:
could we maybe get rid of the SONATYPE_NEXUS_USERNAME/PASSWORD variables now?

Reply:
I had to bring them back because of https://github.com/RevenueCat/purchases-hybrid-common/pull/104

I want to keep that branch with only amazon changes on top of 1.7.1

Reply:
@beylmk @vegaro FYI I ran into some issues with this, and updated it again in #105. I also added the new env variables, but I guess I should probably update the values we're reading here.

Reply:
I see so if I remember correctly, it would fail if it had the env SONATYPE_NEXUS_PASSWORD set even if it was not being accessed right? So I removed them, then that made my Amazon PR fail, so I added them, but that broke your PR lol. So the new version of the plugin doesn't have that issue right?

Reply:
lol
Yeah, or at least, I haven't run into the issue with the newer version... yet. But I didn't see anything in the changelog that would explain it